README.md

```diff
@@ -110,8 +110,8 @@ they do not always need to review the entire EP in depth.
 
 Enhancements should have agreement from all stakeholders prior to
 being approved and merged. Reviews are not time-boxed (see Life-cycle
-below). We manage the rate of churn in the project by asking
-maintainers to include review work along with everything else that
+below). We manage the rate of churn in OKD by asking component
+maintainers to act as reviewers in addition to everything else that
```

Comment:
Going back to my squash merge comment. This is a commit I would suggest folding back in to `add advice to reviewers for improving turn-around time` given you are introducing the verbiage in that commit. PS: I am not going to block on this but just using this to explain my views on commit 😄